aws/cloudformation-templates/services/service/service.yaml

```diff
@@ -300,6 +300,8 @@ Resources:
             - Name: SEGMENT_WRITE_KEY
               ValueFrom: !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/retaildemostore-segment-write-key'
           Environment:
+            - Name: AWS_REGION
```

Comment:
why add this, I didn't see it in the rest of the PR (No real issue, just understanding)

Reply:
Yeah, it's because we're now using the AWS SDK in the microservice and it will fail if it's not given a region.